save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/controller/AgentsController.kt

```diff
@@ -245,20 +246,31 @@ class AgentsController(
     /**
      * @param executionLogs ExecutionLogs
      */
-    @PostMapping("/executionLogs")
-    fun saveAgentsLog(@RequestBody executionLogs: ExecutionLogs) {
+    @PostMapping("/executionLogs", consumes = [MediaType.MULTIPART_FORM_DATA_VALUE])
+    fun saveAgentsLog(@RequestPart(required = true) executionLogs: FilePart) {
+        val fileName = executionLogs.filename()
         val logDir = File(configProperties.executionLogs)
         if (!logDir.exists()) {
             log.info("Folder to store logs from agents was created: ${logDir.name}")
             logDir.mkdirs()
         }
-        val logFile = File(logDir.path + File.separator + "${executionLogs.agentId}.log")
+        val logFile = File(logDir.path + File.separator + "$fileName.log")
         if (!logFile.exists()) {
             logFile.createNewFile()
-            log.info("Log file for ${executionLogs.agentId} agent was created")
+            log.info("Log file for $fileName agent was created")
```

Comment:
```suggestion
            log.info("Log file $fileName was created")
```
Kind of confusing that we assume here that `fileName == agentId`. Or if you want to keep logs like they were previously, maybe just create a separate variable for `agentId`?

Reply:
In our logic file name of log file is equals to `agentId`

However yes, in code it's looks confusing, changed